### PR TITLE
apply ux changes and track events in segment

### PR DESCRIPTION
**Pull request by christianvogt**
## Segment
Sends events to Hybrid Cloud Console segment:

- `DevSandbox Signup Status`: tracked when the user lands on the sandbox page or whenever the user's signup status changes while on the page
- `DevSandbox Signup Start`: tracked when user clicks the `Get Started` button on the landing page
- `DevSandbox Signup Error`: tracked when an errors occurs during the signup process
- `DevSandbox Signup Step`: tracked when the user navigates through the various steps of the signup modal
- `DevSandbox Signup Resend Code`: tracked when the user clicks the `Resend code` button during the signup process
- `DevSandbox Signup Closed`: tracked when the user closes the signup modal
- `DevSandbox Learn More`: tracked when user clicks the learn more link on the landing page
- `DevSandbox Service Launch`: tracked when user clicks a `Launch` button from service catalog
- `DevSandbox Service Learn`: tracked when user clicks a `Learn` button from service catalog

## UX changes
Updated the page according to the latest review changes as well as micro-copy updates.

image

image

Updates to responsive layout:
image
image

### CODE REVIEW

File: src/components/HowItWorksCard/HowItWorksCard.tsx [hunk 1]
```diff
@@ -8,23 +8,23 @@ const HowItWorksCard = () => (
   <Card>
     <CardBody className="how-it-works">
       <TextContent>
-        <Text component={TextVariants.h1}>How it works</Text>
-        <Text>
+        <h1>How it works</h1>
```
Comment (Hyperkid123) [inline src/components/HowItWorksCard/HowItWorksCard.tsx:11]: Any reason why are switching from PF components to the default elements?

File: src/hooks/useTrackEvent.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  event: string;
+  properties?: JSONObject;
+};
+
+export const useTrackEvent = () => {
+  const { analytics } = useChrome();
+  return React.useCallback(
+    (event: string, properties?: JSONObject) => {
+      if (process.env.NODE_ENV !== 'development') {
+        analytics?.track(event, { current_path: window.location.pathname, ...properties });
```
Comment (Hyperkid123) [inline src/hooks/useTrackEvent.ts:20]: We have both dev and prod segment instances. Unless you are developing against the prod environment, you should not need this.

Comment (christianvogt) [reply src/hooks/useTrackEvent.ts:20]: @Hyperkid123 I'm assuming that we use a production build for `prod` and `stage`. The check for `development` is to not send events while developers are running a local dev environment.

File: src/components/HowItWorksCard/HowItWorksCard.tsx (hunk 1 above)
Comment (christianvogt) [reply src/components/HowItWorksCard/HowItWorksCard.tsx]: Updated this, along with other instances, to make use of PF `Text`.

File: src/hooks/useTrackEvent.ts (hunk 2 above)
Comment (Hyperkid123) [reply src/hooks/useTrackEvent.ts:20]: Got it. We can keep this. But just FYI, the do actually filter out RH accounts in the analytics tool (at leas in Pendo). We should not get any noise from developers if that is what you are worried about
.

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**